### DISCUSSION TITLE
My page shows fine locally but on github the theme is gone and the links are broken

### DISCUSSION THREAD

**Discussion by rastiazul** [community/community#22461] [Other Feature Feedback, Questions, & Ideas]
<p>hi, its the first time Im doing this and I followed this guide using jekyll: </p>
<p><a href="https://medium.com/20percentwork/creating-your-blog-for-free-using-jekyll-github-pages-dba37272730a" rel="nofollow noopener">https://medium.com/20percentwork/creating-your-blog-for-free-using-jekyll-github-pages-dba37272730a</a></p>
<p>The website works fine hosted on my localhost but look how broken its on github:<br>
<a href="https://rastiazul.github.io/merino/" rel="nofollow noopener">https://rastiazul.github.io/merino/</a></p>
<p>I bet this must be a common noob error but I couldnt find a solution searching the forum.</p>
<p>Thank you </p>

**Comment by fire-eggs** [comment, marked as answer]
<p>If you look at the page source, the style sheet links to </p>
<pre><code>href="/assets/main.css"
</code></pre>
<p>but that file doesn’t exist, so that is not the location in the github repository.</p>
<p>Either you need a folder named “assets” in the github repository, and main.css in that folder, or you need to fix the web files to point to where the main.css file actually is.</p>

**Reply by genie360s** [reply]
but the markdown is rendered automatically with the workflows , won't the issue in using the gem dependencies to generate the main.css ??

**Comment by reganconnell** [comment]
<p>I’m having the same issue. I’m not sure if your reply applies to my site as well, but I did try to add body tags in my index.html file, it’s not working still.</p>
<p>My site serves right locally, but the github pages version does not show the correct theme, and rather an unstyled website. In my repo’s settings, it says the theme that I want is being used, however, it is not displayed.</p>
<p><a href="https://reganconnell.github.io/regan-connell/" rel="nofollow noopener">https://reganconnell.github.io/regan-connell/</a></p>
<p>Here’s my repo: <a href="https://github.com/reganconnell/regan-connell" rel="nofollow noopener">https://github.com/reganconnell/regan-connell</a></p>

**Reply by catursetiawan** [reply]
hy, I visited your github page. there is going right. can you share solving? I get the same issue as you were. thanks

**Reply by genie360s** [reply]
Sure , 
1. you should make sure the versions of gem dependencies are clearly defined and uptodate , with the one's stipulated by Github .
check here : https://pages.github.com/versions/ 
2. Follow this guide line here : https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site 
3. You might have followed the guideline and still face the same issue ,  
- consider checking your jekyll , minima , github pages and plugins , that the versions correspond to the ones declared  on github , I declared them on the gem file.
- 
![image](https://github.com/community/community/assets/84270692/6cccd181-520b-4c98-8234-be3e09cd4782)

4. Last make sure the deployment branch is gh-pages , and gh-pages is your default branch .
![image](https://github.com/community/community/assets/84270692/d218da61-6808-42ff-b285-e41ea2656259)
you can change it here : 
![image](https://github.com/community/community/assets/84270692/5e358ebc-b2c2-4ed6-8871-03166a1cce4f)
lastly deploy by github actions
![image](https://github.com/community/community/assets/84270692/8ef4e14d-b362-4d24-8b91-dfc05f308f9b)
ensure in your github actions , the deployment branch is also gh-pages

**Comment by adambaileyland** [comment]
<p>Hey Regan, were you able to get this sorted? Your theme looks good now… I’m running into the same .CSS issue on my pages site currently.</p>

**Comment by ekiefl** [comment]
<p>Me too. Did any of the above people fix their problem?</p>

**Comment by billykatz** [comment]
<p>I needed to change my repo name from “me” to “<a href="http://billykatz.github.io" rel="noopener nofollow ugc">billykatz.github.io</a>” That fixed my issues</p>

**Comment by LuqmanQureshi1** [comment]
<p>I tried making my portfolio website by downloading everything from a website and then changing the words and images.<br>
It wasn’t moving up from “your site is ready to be published at” but when I added a theme from GitHub, the site did go up, but it the style was all changed and it was all gibberish.<br>
here is the code: <a href="https://github.com/LuqmanQureshi1/LuqmanTheMarketingStrategistandContentWriter.github.io" rel="noopener nofollow ugc">LuqmanQureshi1/LuqmanTheMarketingStrategistandContentWriter.github.io</a></p>

**Comment by yhammad76** [comment]
https://yhammad76.github.io/The-Dice-game-2/

I have the same problem! any suggestion?

**Comment by jcsonvonyon** [comment]
you may need to check your image directory to be sure it is correct, try this     <link rel="stylesheet" href="../css/fundraising.css">

include two dots and a forward slash if the file is in a subfolder

**Reply by alpserin** [reply]
thank you, this worked for me. change "/assets/main.css" to "../assets/main.css"

**Comment by RaghavGarg1312** [comment]
IN MY CASE THERE WAS A OPEN PULL REQUEST , CLOSED IT AND THE JAVASCRIPT FILE WORKED
